card/VERSION

```diff
@@ -1 +1 @@
-1.14.8s
+1.15.pre
```

Comment:
I do really like that you removed a bunch of copied code.  Wish I'd done it in the first place.  We'll need to be sure all the wagn commands and tasks work still, some don't like references to Card::Anything.